modules/gapi/include/opencv2/gapi/plaidml/gplaidmlkernel.hpp

```diff
@@ -144,4 +146,4 @@ class GPlaidMLKernelImpl: public cv::detail::PlaidMLCallHelper<Impl, typename K:
 
 #endif // OPENCV_GAPI_GPLAIDMLKERNEL_HPP
 
-#endif // if
+#endif // HAVE_PLAIDML
```

Comment:
Unfortunately this doesn't work. As we learned with IE, there must be no conditional compilation in the G-API's public headers. Even if OpenCV is compiled with `HAVE_PLAIDML=1`, user app will not have this flag available.

I'd simply remove this `#ifdef` from this header. Yes, it will depend on some external library - but this is our API as well.

Reply:
fixed